parsons/salesforce/salesforce.py

```diff
@@ -0,0 +1,151 @@
+from simple_salesforce import Salesforce as _Salesforce
+from parsons.utilities import check_env
+# from parsons import Table
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class Salesorce:
+    """
+    `Args:`
+        username: str
+            The Salesforce username (usually an email address).
+            Not required if ``SALESFORCE_USERNAME`` env variable is passed.
```

Comment:
Silly thing, but we are not doing line breaks between these sentences for other classes.

Reply:
But in my case, this will exceed 100 characters. Should I add a `noqa E501`?

Reply:
I was just thinking like a normal line break prior to hitting 100 characters.

```
The Salesforce username (usually an email address). Not required if ``SALESFORCE_USERNAME`` env variable is passed.
```